src/components/NewLocationPage/CompareLocationTabs/CompareLocationTabs.style.tsx

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+import { COLOR_MAP } from 'common/colors';
+import ToggleButton from '@material-ui/lab/ToggleButton';
+import ToggleButtonGroup from '@material-ui/lab/ToggleButtonGroup';
+
+export const ButtonGroup = styled(ToggleButtonGroup)`
+  background-color: ${COLOR_MAP.GREY_1};
+  border: 1px solid ${COLOR_MAP.GREY_2};
+  border-radius: 4px;
+`;
+
+export const Button = styled(ToggleButton)`
+  padding: 0.63rem;
+  border: none;
+  .MuiToggleButton-label {
+    color: ${COLOR_MAP.GREY_3};
```

Comment:
lets darken this to `${COLOR_MAP.GRAY_BODY_COPY}`

Reply:
GREY_3 on the gray background is not accessible (screenshot below, if you use the inspector and hover over the span, youll notice that the contrast is too low)
![Screen Shot 2021-06-09 at 11 13 04 AM](https://user-images.githubusercontent.com/44076375/121381371-aaada380-c913-11eb-94a1-7262344b9a5c.png)

Reply:
👍🏼